pkg/api/accesscontrol.go

```diff
@@ -154,32 +154,30 @@ func (s *Service) actDecryptionHandler() func(h http.Handler) http.Handler {
 // uploads the encrypted reference, history and kvs to the store.
 func (s *Service) actEncryptionHandler(
 	ctx context.Context,
-	w http.ResponseWriter,
 	putter storer.PutterSession,
 	reference swarm.Address,
 	historyRootHash swarm.Address,
-) (swarm.Address, error) {
+) (swarm.Address, swarm.Address, error) {
```

Comment:
Consider using named return values for clarity.
`(encryptedRef, historyRef swarm.Address, err error)`